corebuild/integration/ILLink.Tasks/ILLink.Tasks.targets

```diff
@@ -284,13 +285,30 @@
   <!-- Computes the managed assemblies that are input to the
        linker. Includes managed assemblies from
        ResolvedAssembliesToPublish, and IntermediateAssembly. -->
+  <UsingTask TaskName="FindDuplicatesByMetadata" AssemblyFile="$(LinkTaskDllPath)" />
   <Target Name="_ComputeManagedAssembliesToLink"
           DependsOnTargets="_ComputeManagedResolvedAssembliesToPublish">
     <ItemGroup>
       <_ManagedAssembliesToLink Include="@(IntermediateAssembly)" />
       <_ManagedAssembliesToLink Include="@(_ManagedResolvedAssembliesToPublish)" />
     </ItemGroup>
 
+    <!-- Remove all implementation dlls that have duplicates. This can
+         happen if a portable app has platform-specific managed
+         assemblies. The linker isn't currently able to analyze
+         multiple assemblies with the same identity, so we skip
+         these. -->
```

Comment:
Please update the comment since we no longer skip all duplicates.

Reply:
Updated, thanks!